defaults/main.yml

```diff
@@ -53,23 +53,45 @@ osbs_pruner_old_builds_command_build:
 - "--namespace={{ osbs_namespace }}"
 - builds
 - --orphans=true
-- --keep-younger-than=720h
```

Comment:
It seems suspicious to me that this value is hardcoded in role instead of being configurable from playbook

Reply:
configurable now